features/step_definitions/operators.rb

```diff
@@ -28,9 +28,13 @@
 
 Given /^the status of condition Upgradeable for marketplace operator as expected$/ do
   ensure_admin_tagged
-  actual_status = cluster_operator('marketplace').condition(type: 'Upgradeable', cached: false)['status']
-  status = 'True'
   cluster_version = cluster_version('version').channel.split('-')[1]
+  if cluster_version == "4.1"
+    actual_status = 'True'
```

Comment:
@yuhui-12, I don't see this variable being used anywhere in the step, a typo perhaps?

Reply:
Hi @pruan-rht ,
Thanks a lot for your comments. The variable will be used in line 59.
  unless status == actual_status